EIPS/eip-1.md

```diff
@@ -67,43 +67,29 @@ These calls generally result in a "rough consensus" around what EIPs should be i
 
 ### EIP Process 
 
-Following is the process that a successful non-Core EIP will move along:
+The following is the standardization process for all EIPs in all Tracks:
 
-```
-[ WIP ] -> [ DRAFT ] -> [ LAST CALL ] -> [ FINAL ]
-```
+![EIP Status Diagram](../assets/eip-1/EIP-process.png)
 
-Following is the process that a successful Core EIP will move along:
+**Idea** - An idea that is pre-draft. This is not tracked within the EIP Repository.
 
-```
-[ IDEA ] -> [ DRAFT ] -> [ LAST CALL ] -> [ ACCEPTED ] -> [ FINAL ]
-```
+**Draft** - The first formally tracked stage of an EIP in development. An EIP is merged by an EIP Editor into the EIP repository when properly formatted.
+
+**Review** - An EIP Author marks an EIP as ready for and requesting Peer Review.
+
+**Last Call** - This is the final review window for an EIP before moving to `FINAL`. An EIP editor will assign `Last Call` status and set a review end date (review-period-end), typically 14 days later.
+
+If this period results in necessary material changes it will revert the EIP to `REVIEW`.
+
+**Final** - This EIP represents the final accepted standard. A Final EIP exists in a state of finality and should only be updated to correct errata and add non-material clarifications.
+
+**Stagnant** - Any EIP in `DRAFT` or `REVIEW` if inactive for a period of 6 months or greater is algorithmically moved to `STANGNANT`. An EIP may be resurrected from this state by Authors or EIP Editors through moving it back to`DRAFT`.
```

Comment:
I would prefer 2 months for stagnant.  Stagnant is non-terminal, and I feel like 2 months is *plenty* of time for someone to just *touch* a Draft.

Reply:
```suggestion
**Stagnant** - Any EIP in `DRAFT` or `REVIEW` if inactive for a period of 6 months or greater is moved to `STANGNANT`. An EIP may be resurrected from this state by Authors or EIP Editors through moving it back to`DRAFT`.
```
The wording here is a bit weird, and since we are clearly specifying the condition in the same line I don't think the "algorithmiclly" bit is necessary.

Reply:
I wanted to leave room for a bot or an Editor to make the move as long as the rules are followed. I can change it back to automatically? Or do you mean to leave it out?

Reply:
For the reduction to 2 months. I am fine with this. I am not sure if the change is big enough to need a larger discussion about it.

Reply:
Presumably if/when we add more conditions for moving to stagnant they'll get listed here, or if the list grows too long this will point at that list.  I think just removing the word from the sentence will improve readability and won't cause us any loss of flexibility in the future.